src/csharp/Microsoft.Spark.E2ETest/SparkFixture.cs

```diff
@@ -120,6 +120,7 @@ private void BuildSparkCmd(out string filename, out string args)
             string jarDir = $"{scalaDir}{sep}{jarPrefix}{sep}target";
             string assemblyVersion = Assembly.GetExecutingAssembly().GetName().Version.ToString(3);
             string jar = $"{jarDir}{sep}{jarPrefix}-{assemblyVersion}.jar";
+            string packagesArg = "--packages io.delta:delta-core_2.11:0.3.0";
```

Comment:
Can this be provided from the extension library (so that it is not downloaded by default)? Basically, the core shouldn't know anything about what's happening around it.